app-arch/patool/patool-4.0.2.ebuild

```diff
@@ -103,6 +103,15 @@ python_prepare_all() {
 	sed -e 's/setuptools-reproducible/setuptools/' \
 		-e 's/setuptools_reproducible/setuptools.build_meta/' \
 		-i pyproject.toml || die
+
+	# Workaround setuptools-reproducible setting file stamps to invalid dates.
+	# bug #958470
+	#
+	# Test fails because the timestamp of the zip arhive is before 1980.
```

Comment:
There is a typo here and reproduced in other ebuilds, "arhive" should be "archive".